server/server.go

```diff
@@ -78,35 +78,15 @@ type Config struct {
 // DefaultConfig provides a default configuration to create a new server object
 // by best effort.
 func DefaultConfig() Config {
-	var err error
-
-	var loggerService micrologger.Logger
-	{
-		loggerConfig := micrologger.DefaultConfig()
-		loggerService, err = micrologger.New(loggerConfig)
-		if err != nil {
-			panic(err)
-		}
-	}
-
-	var responderService microtransaction.Responder
-	{
-		responderConfig := microtransaction.DefaultResponderConfig()
-		responderService, err = microtransaction.NewResponder(responderConfig)
-		if err != nil {
-			panic(err)
-		}
-	}
-
 	return Config{
 		// Dependencies.
-		ErrorEncoder:         func(ctx context.Context, serverError error, w http.ResponseWriter) {},
-		Logger:               loggerService,
+		Logger:               nil, // Required.
 		Router:               mux.NewRouter(),
-		TransactionResponder: responderService,
+		TransactionResponder: nil, // Required.
```

Comment:
IMO all dependencies are always required. This is why they are dependencies. The service depends on them.